conduct/code_of_conduct.md

```diff
@@ -102,6 +103,31 @@ free to contact the Code of Conduct Committee at
    automatically imply an admission of guilt.
 
 
+## Responding to inappropriate behavior
+
+In some cases, individuals may violate the CoC in online or in-person situations.
+If any community member deems the action to be harmful to the community,
+they may act immediately to defuse the situation.
+This includes actions directed at the situation, rather than at an individual, such
+as locking a thread, *temporarily* suspending a user's account in the
+community forum, calling a meeting break, or generally engaging in actions
+aimed at de-escalating the situation.
+If the action is ongoing or requires a response directed at a specific individual,
```

Comment:
For several of the actions above, they should _definitely_ be reported rather than allowed to resurface as ongoing.

Reply:
Good point - I don't want to suggest that violations should *generally* need to be ongoing before reporting.

Are there a subset of actions that require formal CoC committee review? Or is it the case that *any* violation of the CoC also requires a review? (this is a question about what the current process is, rather than how it should be - I just don't know the answer).

Reply:
actually reading back through my original comment on this PR (sorry it's been a few months so I haven't thought about this in a while), I think any "action" taken in response to a CoC violation should be reported, so perhaps the language about "ongoing" should just be removed?

Here I am thinking from the perspective of a moderator in the community forum - if I lock a thread because somebody is being a jerk, should I report that to the CoC each time? And would that be different from reporting CoC violation that requires a formal investigation etc (or perhaps that is purely for the CoC committee to decide after the report?)

Reply:
Yeah I think removing "ongoing" is good.